instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/patches/client.rb

```diff
@@ -68,11 +63,12 @@ def query(sql)
           def client_attributes(sql)
             attributes = {
               ::OpenTelemetry::SemanticConventions::Trace::DB_SYSTEM => 'mysql',
-              ::OpenTelemetry::SemanticConventions::Trace::NET_PEER_NAME => net_peer_name
+              ::OpenTelemetry::SemanticConventions::Trace::NET_PEER_NAME => connection_options.fetch(:host, 'unknown sock'),
             }
 
             attributes[::OpenTelemetry::SemanticConventions::Trace::DB_NAME] = database_name if database_name
             attributes[::OpenTelemetry::SemanticConventions::Trace::PEER_SERVICE] = config[:peer_service] unless config[:peer_service].nil?
+            attributes['db.mysql.instance.host.name'] = @connected_host if defined?(@connected_host)
```

Comment:
I have no idea what this be named and I am struggling here. 

What do you think @adrianna-chang-shopify @cschleiden ?

Reply:
cc: @open-telemetry/specs-semconv-approvers I could use some of your input/feedback here as well.

Reply:
I see https://github.com/open-telemetry/opentelemetry-ruby/blob/main/semantic_conventions/lib/opentelemetry/semantic_conventions/trace.rb#L71 

looking at https://github.com/open-telemetry/opentelemetry-specification/pull/3402, maybe
`db.mysql.instance.address` or `db.mysql.address`? But `db.mysql.instance.host.name` would also work for me

Reply:
Nice find!

Reply:
cc: @andyedison